### ISSUE TITLE
[Feature request] 60 FPS patches added alongside widescreen patches

### ISSUE THREAD

**Issue by vgturtle127** [PCSX2/pcsx2#1819]
I was curious how you guys would feel about adding 60 FPS patches in addition to the widescreen patches. I'm not sure if you would need separate .pnach files, or if you could combine them into one (not sure that would be optimal for testing and stuff), or if you would need to do something altogether different. Adding a GUI option to enable 60 FPS patches would be good as well.

It should be noted some of the codes haven't been fully tested, but most seem to work fine.

The community has made much of the PS2 library already available at 60 FPS [HERE](http://forums.pcsx2.net/Thread-60-fps-codes), so the work wouldn't be entirely on us.

Let's discuss, but I think if widescreen was added, 60 FPS probably makes sense as well.

**Comment by avih** [comment]
> I was curious how you guys would feel about adding 60 FPS patches in addition to the widescreen patches.

Is there an archive of all the patches? How many are there roughly so far? How many of them perform as expected with minimal issues (other than more load)?

Also, what do you mean by "adding"? the widescreen patches "support" has few parts to it:
- A curator which manages the patches with the community, rejects bad ones, sorts the list to patches which are "release worthy" and those which are not, update the collection, etc. Currently Devina does this for the widescreen patches.
- Config/GUI to control whether or not to enable widescreen patches, and when it's enabled:
- Support for loading patches from the dir "cheats_ws".
- Support for loading patches from `cheats_ws.zip`. The zip is a curated (by Devina) set of patches which we periodically clean a bit and then add to the repo to be distributed with PCSX2.

The only way I can interpret "adding" is to duplicate all of those also for 60fps patches, which, TBH, will be a bit of a PITA. This also includes someone who can curate and manage the set of patches, reject or remove buggy ones, etc, as this is not something the PCSX2 team has resources to engage in.

It's really important to have a curated list since if it's distributed with PCSX2 and users can enable it, it must go through some quality process rather than be a random collection of patches with varying quality.

The bottom line, IMHO, is that it mostly depends on how many quality 60fps patches are there, how many users use them or want to use them, and that there's someone who is willing to commit to curate and manage this set of patches.

I don't think PCSX2 should add support for those patches (with UI etc - obviously they can work as regular patches already) before it becomes a meaningful enough thing which is already well managed.

And then of course you'd also need someone who is willing and capable of adding the actual code to PCSX2 itself.

**Comment by vgturtle127** [comment]
@avih Here is a compiled list of all of the patches:

[Link](https://www.reddit.com/r/emulation/comments/5tmge2/ps2_60fps_codes_list/
)

A majority of PCSX2 enthusiasts are using them, and have tested them pretty extensively without many issues. A majority of the work is done by the user asasega on the forums.

**Comment by FlatOutPS2** [comment]
These patches are very much unfinished, untested and as @Ryudo300 pointed out on the wiki, also often require adjusting PCSX2 to not make it run at 200% speed. This is not something that should be added to PCSX2. As far as I'm aware no other emulator that has these kinds of patches includes 'em in an archive, they're too volatile.
